src/bin/utils/parsers.ts

```diff
@@ -56,4 +61,87 @@ function parseIdentityString(identityString: string): {
   return { providerId, identityId };
 }
 
-export { parseNumber, parseSecretPath, parseGestaltId };
+/**
+ * Acquires the default seed nodes from src/config.ts.
+ */
+function getDefaultSeedNodes(): NodeMapping {
```

Comment:
In the `src/config.ts`, which should have `config.defaults.network.testnet`. We have to select between `config.defaults.network.testnet` or ``config.defaults.network.mainnet` based on an environment variable.

This environment variable can be `PK_NETWORK`. This can be set to either `"testnet"` or `"mainnet"`.

Additionally this can be added as a CLI parameter. Like `--network`. This would only be needed for `pk agent start`. So imagine `pk agent start --network=mainnet`.

When the network parameter is incorrect, we would give back an exception. That exception can be `commander.InvalidArgumentError`. See how this is done in `parseNumber`.

Therefore:

```ts
const network = new commander.Option(
  '-n, --network <network>',
  'Network'
)
  .env('PK_NETWORK')
  .argParser(binParsers.parseNetwork)
  .default(config.defaults.network.mainnet);
```

Reply:
- Added the network option. this is used in agent start.
- Added an `network` argument to `getDefaultSeedNodes` to select witch network is the default.